cpp/tensorrt_llm/kernels/cutlass_kernels/include/moe_kernels.h

```diff
@@ -926,5 +926,36 @@ struct GemmProfilerBackend
 // Populates a buffer with random values for use with MOE benchmarking
 void populateRandomBuffer(void* buffer_void, size_t size, cudaStream_t stream);
 
+bool fusedBuildExpertMapsSortFirstToken(int const* token_selected_experts, int* unpermuted_token_selected_experts,
```

Comment:
Lets create a separate header for these changes. This current header is shared with the internal cutlass kernels at the moment so adding them here could cause some issues for that option

Reply:
Done.